# Nanopore Modified Basecalling Pipeline

## Goal 
Our goal is to examine the epigenetic response of animals to heat stress compared to those in normal conditions. To accomplish this, we'll develop a bioinformatics pipeline that utilizes Nanopore's MinION technology to detect methylation patterns. We'll also conduct a PCR-based bisulfite assay to determine methylation patterns in specific genes. Finally, we'll test the accuracy of both methods to ensure consistent results or to test for general congruency. This GitHub repository is specifically for the nanopore aspect of our project. Here, you can find a detailed protocol for a possible bioinformatics timeline, outlined step-by-step.  

## General Pipeline
 
1. Obtain Raw Fast5 files produced from MinION
2. Regular Basecalling using Guppy 
4. Perform Modified Basecalling using either program
   - Megalodon
   - Nanopolish (https://nanopolish.readthedocs.io/en/latest/quickstart_call_methylation.html)
   - [See reference for program choice:] (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8524990/)

  
## Nanopolish Pipeline

<img width="642" alt="image" src="https://github.com/dys0004/FitBirds/assets/29266059/322b1dc6-11e7-4a4f-a24a-5f6a2b17724c">




   
![zebra_finch_animation_by_nashiil_gif (536Ã—459)](https://github.com/dys0004/FitBirds/assets/29266059/e16b149c-37ca-4e6d-96c5-026dc8a25088)
